kiwi/volume_manager/btrfs.py

```diff
@@ -413,19 +407,24 @@ def sync_data(self, exclude=None):
         """
         if self.toplevel_mount:
             sync_target = self.get_mountpoint()
-            if self.custom_args['root_is_snapper_snapshot']:
-                self._create_snapshot_info(
-                    ''.join(
-                        [
-                            self.mountpoint,
-                            f'/{self.root_volume_name}/.snapshots/1/info.xml'
-                        ]
-                    )
-                )
             data = DataSync(self.root_dir, sync_target)
             data.sync_data(
                 options=Defaults.get_sync_options(), exclude=exclude
             )
+            if self.custom_args['root_is_snapper_snapshot']:
+                root_prefix = os.path.join(
+                    self.mountpoint,
+                    f'{self.root_volume_name}/.snapshots/1/snapshot'
```

Comment:
Can we have snapper tell us what this is? This is still an assumption of how snapper works rather than snapper telling us stuff.